src/components/safe-apps/SafeAppIcon/index.tsx

```diff
@@ -4,7 +4,7 @@ const APP_LOGO_FALLBACK_IMAGE = `/images/apps/app-placeholder.svg`
 
 const getIframeContent = (url: string, width: number, height: number): string => {
   return `
-     <body style="margin: 0;">
+     <body style="margin: 0; overflow: hidden;">
```

Comment:
@mikhailxyz display block on the image would make this fix unnecessary.

Reply:
absolutely insane stuff